src/components/ArcadeMode.jsx

```
@@ -0,0 +1,548 @@
+              <div className="battle-text">
+                <h1>FIGHT!</h1>
+                <div className="battle-subtitle">Let the battle begin!</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      )
+
+    case 'victoryMoment':
+      const latestScore = matchScores[matchScores.length - 1] || 0
+      return (
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)